api_v3/lib/KalturaLiveEntryService.php

```diff
@@ -141,7 +132,10 @@ function appendRecordingAction($entryId, $assetId, $mediaServerIndex, KalturaDat
 			$dbEntry->save();
 		}
 
-		kJobsManager::addConvertLiveSegmentJob(null, $dbAsset, $mediaServerIndex, $filename, $currentDuration);
```

Comment:
I would move the parsing into the convert-live-segment to be done asynchronously on the batch side.